.travis.yml

```diff
@@ -30,20 +30,8 @@ cache:
   - $HOME/.ivy2/cache
   - $HOME/.ivy2
 
-before_install:
```

Comment:
I'm skipping these sections in Travis because we're removing them in incoming PRs:

https://github.com/frees-io/freestyle-rpc-examples/issues/3